assets/wire/components/ItemsList.jsx

```diff
@@ -102,14 +103,15 @@ class ItemsList extends React.Component {
     }
 
     render() {
-        const {items, itemsById, activeItem} = this.props;
+        const {items, itemsById, activeItem, selectedItems, readItems} = this.props;
         const filteredActions = this.filterActions();
         const articles = items.map((_id) =>
             <WireListItem
                 key={_id}
                 item={itemsById[_id]}
                 isActive={activeItem === _id}
-                isSelected={this.props.selectedItems.indexOf(_id) !== -1}
+                isSelected={selectedItems.indexOf(_id) !== -1}
+                isRead={readItems[_id] === (parseInt(itemsById[_id].version, 10) || 0)}
```

Comment:
would probably add another util function to keep this parseInt handling in one place